lib/Payout/Connection.php

```diff
@@ -131,9 +131,9 @@ public function authenticate($url, $client_id, $client_secret)
     public function post($url, $body, $idempotency_key)
     {
         $this->addHeader('Authorization', 'Bearer ' . $this->token);
-        
-        if (!is_null($idempotency_key)) {
-           $this->addHeader('idempotency-key', $idempotency_key);
+
+        if ($idempotency_key !== null) {
+            $this->addHeader('idempotency-key', $idempotency_key);
```

Comment:
na toto je nejaky specificky dovod? nie je funkcia is_null totozna a podla mna aj rychlejsia a citatelnejsia?

Reply:
Funkcionalita je totožná áno, práve preto je použitie funkcie pomalšie. Ten rýchlostný rozdiel je ale samozrejme zanedbateľný.
Jediný rozdiel bude v čitateľnosti. Zmenil som to kvôli osobnej preferencii, ale ak preferujete funkciu, zmením to späť.

Celý dôvod prečo mi vôbec napadlo to zmeniť je, že vo vašej pôvodnej knižnici tento úryvok kódu nie je a bol pridaný niekedy dávno do tohto pluginu. Preto keď som ju updatol a prechádzal zmeny, tak mi napadlo toto zmeniť.

Reply:
<img width="622" alt="CleanShot 2023-05-10 at 11 56 50@2x" src="https://github.com/payout-one/payout_woocommerce/assets/65039589/88eabe3e-c6f7-4344-861b-2c3490719071">
ja len ze asi sa to uz zmenilo aj ked php som priamo neprogramoval 10r

Reply:
Máte pravdu, už to je asi naopak, prehodím to späť na funkciu.